definitions/v7/redbits-api.yml

```diff
@@ -6,7 +6,7 @@ language: es-ES
 type: private
 encoding: UTF-8
 links:
-  - https://red-bits.com/
+  - https://redbits.xyz/
```

Comment:
you also need to add the old url as a legacy link so it migrates the existing url for users.